UI/CSS/orders.css

```diff
@@ -0,0 +1,101 @@
+@media only screen and (max-width: 640px) {
```

Comment:
Are you doing responsiveness for only those screens or is there more... forexample those small iphone screens...
any way responsiveness exceeds expectation and it good you are exceeding expectations...

Reply:
I am planning to add in more screen sizes with time....

> Phiona have ue hosted with gh-pages, if have, you put the link in the description so one can easily look at ur design